server/graphql/v2/object/AccountStats.js

```diff
@@ -80,6 +82,36 @@ export const AccountStats = new GraphQLObjectType({
           return collective.getTotalAmountReceivedAmount({ kind, startDate: args.dateFrom, endDate: args.dateTo });
         },
       },
+      totalPaidExpense: {
```

Comment:
We need some consistency on plural vs singular. If other function say "paidExpenses", we should be plural here. Or the opposite.